selectel/resource_selectel_mks_nodegroup_v1.go

```diff
@@ -154,6 +154,11 @@ func resourceMKSNodegroupV1() *schema.Resource {
 				Computed:     true,
 				RequiredWith: []string{"enable_autoscale", "autoscale_min_nodes"},
 			},
+			"nodegroup_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
It seems that `nodegroup_type` should have `ForceNew: true` property.
Also It would be great if we add validation func for this values.